src/docfx/restore/RestoreFlags.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Docs.Build
+{
+    [Flags]
+    internal enum GitFlags
```

Comment:
name does not match file name

Reply:
my mistake, but why no warnings reported?